android/build.gradle

```diff
@@ -2,17 +2,18 @@
 
 buildscript {
     ext {
-        buildToolsVersion = "29.0.2"
-        minSdkVersion = 19
+        buildToolsVersion = "29.0.3"
+        minSdkVersion = 21
         compileSdkVersion = 29
         targetSdkVersion = 29
+        ndkVersion = "20.1.5948944"
     }
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath('com.android.tools.build:gradle:4.2.1')
+        classpath("com.android.tools.build:gradle:4.1.0")
```

Comment:
@limkimsan why this downgrade version?

Reply:
react-native version 0.64.1 is not supported with Gradle version 4.2.1. I reason that I upgrade the Gradle version to 4.2.1 in the previous code is to be able to generate Native debug symbols file to upload to Google Play Console.